tests/test_cachedataset_persistent_workers.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from monai.data import (
+    create_test_image_2d,
+    CacheDataset,
+    DataLoader,
+)
+from monai.transforms import (
+    Compose,
+    RandAffined,
+    Spacingd
+)
+
+
+class TestTransformsWCacheDatasetAndPersistentWorkers(unittest.TestCase):
```

Comment:
Please add this test to the min_tests list to skip.

Reply:
Hi @rijobro ,

I think this PR is important to enhance the `CacheDataset`, can you please help run some tests for speed and memory,  then let's merge it?

Thanks.

Reply:
Sure I'll have a look now.

Reply:
Confirmed as same execution time and same results on the spleen_segmentation_3d notebook.